metadata-io/src/main/java/com/linkedin/metadata/search/EntityLineageResultCacheKey.java

```diff
@@ -16,28 +14,20 @@ public class EntityLineageResultCacheKey {
   private final Long startTimeMillis;
   private final Long endTimeMillis;
   private final Integer maxHops;
+  private final TemporalUnit temporalUnit;
```

Comment:
Storing the `TemporalUnit` causes our tests to fail with:
```
Caused by: com.hazelcast.nio.serialization.HazelcastSerializationException: The 'interface java.time.temporal.TemporalUnit' cannot be serialized with zero configuration Compact serialization because this type is not supported yet. If you want to serialize 'class com.linkedin.metadata.search.EntityLineageResultCacheKey' which uses this class in its fields, consider writing a CompactSerializer for it.
```

cc @RyanHolstien

Reply:
I'll be removing the temporal unit to have a green CI, unless you have a reason to keep it?

Reply:
Yeah no reason to keep it